### PR TITLE
add proc_net_ip_vs_stats charts info

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary

This PR adds dashboard info for [proc_net_ip_vs_stats](https://github.com/netdata/netdata/blob/master/collectors/proc.plugin/proc_net_ip_vs_stats.c) collector's charts.

##### Component Name

`web/`

##### Test Plan

Install this PR; check the proc_net_ip_vs_stats charts; ensure the info is on the dashboard.

##### Additional Information

Fixes part of netdata/product#2104

### CODE REVIEW

**Review by thiagoftsm — DISMISSED** [review]

File: web/gui/dashboard_info.js [hunk 1]
```diff
@@ -1724,6 +1732,19 @@ netdataDashboard.context = {
         ]
     },
 
+    // ------------------------------------------------------------------------
+    // IPVS
+    'ipvs.sockets': {
+        info: 'Summary connection rate of all services and their servers. '+
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:1738]:
```suggestion
        info: 'Total number of connections for all services and their servers. '+
```

File: web/gui/dashboard_info.js [hunk 2]
```diff
@@ -1724,6 +1732,19 @@ netdataDashboard.context = {
...
+        'To see the IPVS connection table, run <code>ipvsadm -Lnc</code>.'
+    },
+    'ipvs.packets': {
+        info: 'Summary packet rate of all services and their servers.'
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:1742]:
```suggestion
        info: 'Total number of received and sent packets for all services and their servers.'
```

File: web/gui/dashboard_info.js [hunk 3]
```diff
@@ -1724,6 +1732,19 @@ netdataDashboard.context = {
...
+    },
+    'ipvs.net': {
+        info: 'Summary throughput of all services and their servers.'
```
Comment (vlvkobal) [inline web/gui/dashboard_info.js:1745]:
```suggestion
        info: 'Total amount of data received and sent for all services and their servers.'
```

Comment (ilyam8) [reply web/gui/dashboard_info.js:1745]: Not amount, that is throughput

File: web/gui/dashboard_info.js (hunk 2 above)
Comment (ilyam8) [reply web/gui/dashboard_info.js:1742]: That counter is packet rate

File: web/gui/dashboard_info.js (hunk 1 above)
Comment (ilyam8) [reply web/gui/dashboard_info.js:1738]: That counter is connection rate

**Review by vlvkobal — DISMISSED** [review]

**Review by vlvkobal — DISMISSED** [review]

**Review by thiagoftsm — APPROVED** [review]

**Review by vlvkobal — APPROVED** [review]

**Pull request merged**